### PR TITLE
Display test progress

### PR DESCRIPTION
**Pull request by arcusfelis**
Proposed changes include:
* Print + when test passes
* Print - when test fails
* Force sample_external_auth.py to use python2 (on my machine it tries to use python3 and fails)

### CODE REVIEW

File: apps/ejabberd/src/ct_progress_hook.erl [hunk 1]
```diff
@@ -0,0 +1,67 @@
+%%% @doc Common Test Example Common Test Hook module.
+-module(ct_progress_hook).
```
Comment (michalwski) [inline apps/ejabberd/src/ct_progress_hook.erl]: Can we have this module in `apps/ejabberd/test` instead of `src`? It better suits there.

Comment (arcusfelis) [inline apps/ejabberd/src/ct_progress_hook.erl]: We can't. rebar is too stupid to compile modules in test.

**Comment by arcusfelis** [comment]
It looks like this https://travis-ci.org/esl/MongooseIM/jobs/152645125#L406

**Comment by michalwski** [comment]
Is it possible to silence log messages like the one here: https://travis-ci.org/esl/MongooseIM/jobs/152645125#L488 ?

**Pull request merged**